### PR TITLE
Added 6 New Resources README

### PR DESCRIPTION
**Pull request by jennvlasiu**
1. General Blog/Articles/Applications: Added new WEF visualization resource that uses twitter API and NLP to monitor topic & sentiment evolution on global matters  

2. Tools: Delated Peter Holme's word stemmer as link no longer appears to work (http://holme.se/stem/)

3. Tools: Added new lemmatization & stemming tool for visualizing 17 languages supported by the NLTK 2.0.4 stem package

4. Tools: Added MIT OpenNPT for neural machine translation and neural sequence modeling open source 

5. Products: Added Systran - Machine Translation Solution for Enterprise

6. Biases in NLP: Proposing new section to help foster collaboration and create a shared space to shed light on ethical/societal implications in NLP developments (i.e. associations offensively skewed, biased word embeddings etc)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +68,7 @@ A curated list of resources for learning about natural language processing, text
 ## Blog Articles, Papers, Case Studies
 
 ### General
+- [WEF Live Campaign - Twitter fed Global News Topics & Sentiment Tracker] (https://365.weflive.com/#!/r/worldwide/t/all-topics) - Live Jan 2019
```
Comment (stepthom) [inline README.md]: You have a space between the `[]` and `()` brackets, which causes these links to not be rendered correctly in the markdown. Please change them all to this format: 

```suggestion
- [WEF Live Campaign - Twitter fed Global News Topics & Sentiment Tracker](https://365.weflive.com/#!/r/worldwide/t/all-topics) - Live Jan 2019
```

File: README.md [hunk 2]
```diff
@@ -99,6 +99,9 @@ A curated list of resources for learning about natural language processing, text
 - [A Tidytext Analysis of the Weinstein Effect](https://www.gokhanciflikli.com/post/weinstein-effect/) Dec 2013.
 - [Comparison of the Most Useful Text Processing APIs](https://www.kdnuggets.com/2018/08/comparison-most-useful-text-processing-apis.html)
 
+### Biases in NLP 
+- [Thought Leadership Blogpost-Biases in Word Embeddings] (https://neurodivergent-ai.github.io/blog/post_1.html) - Natural Language Processing News Dec 2018, Issue #35.
```
Comment (stepthom) [inline README.md]: I'm a little confused here. The post you link to is titled "what words mean" and is from a blog titled "neurodivergent ai". How do "Though Leadership Blogpost-Biases in Word Embeddings" and "Natural Language Processing News Dec 2018, Issue #35." relate? Please clarify.

Comment (jennvlasiu) [reply README.md]: Apologies for any confusion. The blogpost was initially discovered via the NLP Newsletter that was associated with NeurIPS 2018.

However for clarity, I have proposed a different blogpost/thought leadership article discussing gender biases in datasets and word embeddings. Hopefully illustrates the topic more clearly.

**Comment by stepthom** [comment]
@jennvlasiu Thank you! Looks great. 👍

**Pull request merged**